test/unit-tests.ts

```diff
@@ -0,0 +1,23 @@
+import * as assert from "assert";
+import * as functions from "./../src/functions/functions";
+import * as mocha from "mocha";
+
+describe("Test Excel Custom Functions", function () {
```

Comment:
Consider naming this "Unit Tests".

Reply:
Are you saying "Unit Tests" instead of "Test Excel Custom Functions"?

Reply:
yes